openedx/features/idea/tests/test_backend_storage.py

```diff
@@ -0,0 +1,74 @@
+import os.path
+
+from boto import connect_s3
+from boto.s3.key import Key
+from django.core.files.base import ContentFile
+from django.test import TestCase
+from django.test.utils import override_settings
+from mock import patch
+
+from common.test.utils import MockS3Mixin
+
+from ..backend_storage import CustomS3Storage
+
+
+@override_settings(MEDIA_ROOT='/tmp')
+class CustomS3StorageTest(MockS3Mixin, TestCase):
+
+    @patch.object(CustomS3Storage, 'aws_bucket_name', 'test_bucket')
+    @patch.object(CustomS3Storage, 'aws_access_key_id', 'test_key')
+    @patch.object(CustomS3Storage, 'aws_secret_access_key', 'test_key')
```

Comment:
Can't we avoid this by have these settings in test settings file?

Reply:
I will move keys to configurations but there is a possibility of breaking `edx` test, since they do not have these values in test configurations